lumigator/frontend/src/stores/datasetsStore.ts

```diff
@@ -56,6 +56,9 @@ export const useDatasetStore = defineStore('datasets', () => {
     }
     inferenceJobs.value = allJobs
       .filter((job) => job.metadata.job_type === 'inference')
+      // NOTE: 'temporary fix' to prevent showing inference jobs that weren't created by the UI
+      // to generate ground truth.
+      .filter((job) => job.name.startsWith("Ground truth for "))
```

Comment:
I think the `jobs` endpoint can take a `types` query param now, we can just ask for annotation jobs only from the backend

Reply:
It seems we currently have a bug in listing jobs filtering by annotate job type: https://github.com/mozilla-ai/lumigator/issues/989 😢